WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -648,6 +653,49 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
         mediaInserterHelper.cancelUploadOf(media: media)
     }
 
+    func gutenbergDidRequestToSetFeaturedImage(for mediaID: Int32) {
+        // dismiss current controller in order to allow the new alert controller to be presented
+        presentedViewController?.dismiss(animated: false, completion: nil)
+
+        let featuredImageId = post.featuredImage?.mediaID
+        let mediaIdNoFeaturedImageSet = 0
```

Comment:
This seems to be declared in two places ([this](https://github.com/wordpress-mobile/WordPress-iOS/pull/16426/files#diff-1f88390d90a5eb0598cd4101f287ac4468bfe3229fa20eb41e78d386efc6fa28R8) being the other).
Would it make sense to put this in a static constant inside `GutenbergFeaturedImageHelper`, like [this](https://github.com/wordpress-mobile/WordPress-iOS/blob/6bd1fbfeae283cc43e04acbf40386b87051fe4aa/WordPress/Classes/ViewRelated/User%20Profile%20Sheet/UserProfileSiteCell.swift#L11)?

Reply:
Ah, that makes a lot of sense, I can see how that would be better practice! I've gone ahead to make that change in https://github.com/wordpress-mobile/WordPress-iOS/pull/16426/commits/9d1aaf599e76370edcb0fedcc3d1077e9e8ee590. 🙇‍♀️